src/index.js

```diff
@@ -1,11 +1,30 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import ReactDOM from 'react-dom';
 
-import {
-  App
-} from './components';
+import { ChakraProvider } from '@chakra-ui/react';
+import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';
+import { Header, Catalog } from './components';
 
-ReactDOM.render(
-  <App />,
-  document.getElementById('root')
-);
\ No newline at end of file
+const App = () => {
+	return (
```

Comment:
Maybe add another `callApi` function here, and in here, it calls the real API, and sets the status accorgly.